components/bl-audio-recorder/src/index.js

```diff
@@ -23,6 +23,7 @@ export default function AudioRecorder({ component, eventHandlers, elRef }) {
 
   const timer = useMemo(() => new Timer(setTime), []);
   const isRecording = useMemo(() => state && state !== StreamState.INACTIVE, [state]);
+  const isPaused = useMemo(() => state === StreamState.PAUSED, [state]);
```

Comment:
No sense in this memo, it's pretty simple calculations and memo make them more difficult